config/asus-zenfone3.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+OPTIONS=(-r vsn -r vsp)
+PANELS=(
+	[ctc5p5_ili7807b_1080p_video]="asus,ctc5p5_ili7807b"
```

Comment:
```suggestion
	[ctc5p5_ili7807b_1080p_video]="asus,zs520kl-ili7807b-cts5p5"
```

Reply:
wait, what is this for?

btw "5p2" is for ze520kl, while "5p5" is for ze552kl

Reply:
You can update lk2nd, than use ze552kl and ze520kl prefix and remove the post fixes

Reply:
or you can used postfixes without 5p5 and 5p2

Reply:
hmm... wdyt about this commit? looks good?
https://github.com/Smankusors/lk2nd/commit/8f33f760fad7523b2889c20e98c7a8d6901c9764

Reply:
yes it seems fine